quic/s2n-quic-core/src/recovery/hybrid_slow_start.rs

```diff
@@ -61,8 +98,14 @@ impl HybridSlowStart {
         time_of_last_sent_packet: Timestamp,
         rtt: Duration,
     ) {
-        if congestion_window >= self.threshold {
-            // We are already out of slow start so nothing to do
+        if congestion_window >= self.threshold
+            || (self.use_hystart_plus_plus && self.threshold < f32::MAX)
```

Comment:
this would make it a little more clear in my view: 

```suggestion
        let ss_threshold_found = self.threshold < f32::MAX;
        if congestion_window >= self.threshold
            || (self.use_hystart_plus_plus && ss_threshold_found)
```

Reply:
Got it. thanks for the suggestion.